frontend/src/components/shared/FullScreenWrapper/FullScreenWrapper.scss

```diff
@@ -0,0 +1,32 @@
+.streamlit-container .stFullScreenFrame--expanded {
+  position: fixed;
+  top: 0px;
+  left: 0px;
+  bottom: 0px;
+  right: 0px;
+  overflow: auto;
+  background: white;
+  z-index: 1000;
+  padding: 30px;
+  padding-top: 80px;
+}
+
+.streamlit-container .stFullScreenFrame .stButton-enter {
+  position: absolute;
+  right: 10px;
+  top: 10px;
```

Comment:
Look at header.scss and see what we use for the hamburger menu, make it a variable and use it here.

(Also, I think we don't use px as a unit for this things, but I may be mistaken...)